.infra/workers.ts

```diff
@@ -23,18 +23,10 @@ export const workers: Worker[] = [
     topic: 'views',
     subscription: 'add-views-v2',
   },
-  {
```

Comment:
Did you check if these topics can also be removed now?
(not sure if they have other subscriptions)

Reply:
Ah felt like it could be a optimization cleanup for pushing to topics that have no subscriber.
But def not as important.